src/test/java/core/basesyntax/tests/TestStorage.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+public class TestStorage {
+    private static Map<Fruit, Integer> data = new HashMap<>();
+    private static Map<Fruit, Integer> storage = Storage.getFruits();
```

Comment:
this test is not quite informative - you actually compare two java.util.Maps

there is no business logic in Storage class